ui/app/ducks/swaps/swaps.js

```diff
@@ -452,21 +451,21 @@ export const signAndSendTransactions = (history, metaMetricsEvent) => {
     const usedGasPrice = customConvertGasPrice || tradeTxParams?.gasPrice || fastGasEstimate
     usedTradeTxParams.gasPrice = usedGasPrice
 
-    const conversionRate = getConversionRate(state)
+    const usdConversionRate = getUSDConversionRate(state)
     const destinationValue = calcTokenAmount(usedQuote.destinationAmount, destinationTokenInfo.decimals || 18).toPrecision(8)
     const usedGasLimitEstimate = usedQuote?.gasEstimateWithRefund || (`0x${decimalToHex(usedQuote?.averageGas || 0)}`)
     const totalGasLimitEstimate = (new BigNumber(usedGasLimitEstimate, 16)).plus(usedQuote.approvalNeeded?.gas || '0x0', 16).toString(16)
-    const gasEstimateTotalInEth = getValueFromWeiHex({
+    const gasEstimateTotalInUSD = getValueFromWeiHex({
       value: calcGasTotal(totalGasLimitEstimate, usedGasPrice),
       toCurrency: 'usd',
-      conversionRate,
+      conversionRate: usdConversionRate,
       numberOfDecimals: 6,
     })
     const averageSavings = usedQuote.isBestQuote
       ? decEthToConvertedCurrency(
         usedQuote.savings?.total,
         'usd',
-        conversionRate,
+        usdConversionRate,
```

Comment:
While not a `gas_fees` property, the `averageSavings` calculation was also incorrectly using on the `conversionRate`, when we want our analytics in USD